Makefile

```diff
@@ -214,6 +225,28 @@ Switzerland North:cldrswitzerlandnorth,\
 Germany West Central:cldrgermanywestcentral,\
 Norway East:cldrnorwayeast
 endef
+
+ifndef AZURE_STORAGE_ACCOUNTS
+ifdef IMAGE_COPY_PHASE
```

Comment:
You can simplify L230-L237 with the following:
`if ($(IMAGE_COPY_PHASE),test)
AZURE_STORAGE_ACCOUNTS=$(AZURE_STORAGE_ACCOUNTS_TEST_PHASE)
else
AZURE_STORAGE_ACCOUNTS=$(AZURE_STORAGE_ACCOUNTS_ALL)
endif
`

Its not needed to check IMAGE_COPY_PHASE to be defined as you define it with default value in L14